bpf/http_sock.c

```diff
@@ -376,8 +376,8 @@ int socket__http_filter(struct __sk_buff *skb) {
             }
             read_skb_bytes(skb, tcp.hdr_len, info.buf, full_len);
             u64 cookie = bpf_get_socket_cookie(skb);
-            bpf_dbg_printk("=== http_filter cookie = %llx, tcp_seq=%d len=%d %s ===", cookie, tcp.seq, len, buf);
-            dbg_print_http_connection_info(&conn);
+            //bpf_dbg_printk("=== http_filter cookie = %llx, tcp_seq=%d len=%d %s ===", cookie, tcp.seq, len, buf);
```

Comment:
Is this commented code intentionally?

Reply:
Yes, it seems various of these bpf printk statements cause issues on older kernels. Not all though, but I've never been able to figure out when and why. We get unknown func, but it works on newer kernels. I think it's old verifier bugs.